addons/website/models/website_menu.py

```diff
@@ -139,24 +141,47 @@ def _clean_url(self):
         return url
 
     def _is_active(self):
+        """ To be considered active, a menu should either:
+
+        - have its URL matching the request's URL and have no children
+        - or have a children menu URL matching the request's URL
+
+        Matching an URL means, either:
+
+        - be equal, eg ``/contact/on-site`` vs ``/contact/on-site``
+        - be equal after unslug, eg ``/shop/1`` and ``/shop/my-super-product-1``
+
+        Note that saving a menu URL with an anchor or a query string is
+        considered a corner case, and the following heuristic will be applied:
```

Comment:
Nitpicking: "the following applies:" instead of "the following heuristic will be applied:". It kinda was an heuristic before but not anymore IMO :grimacing: